### PR TITLE
added slider to home page with padding

### PR DESCRIPTION
**Pull request by PietyE**
**Add slider to homepage (https://trello.com/c/QdoyZ7my/3-creat-news-slider-at-home-page)**
-added slider to home page

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -3,11 +3,13 @@ import { Router } from './components/Router';
 import { Header } from './components/Header';
 import { ThemeProvider } from '@mui/material';
 import { theme } from './theme/theme';
+import { Slider } from './components/Slider';
 
 export const App = () => {
   return (
     <ThemeProvider theme={theme}>
       <Header />
+      <Slider />
```
Comment (kerrimov) [inline src/App.tsx:12]: you should use `Home` component

Comment (PietyE) [reply src/App.tsx]: fixed

File: src/components/Slider/Slider.tsx [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
-import { Autoplay, Navigation } from 'swiper';
+import SwiperCore, { Autoplay, Navigation } from 'swiper';
 import { Container } from '@mui/material';
 import { SliderCard } from './components/SliderCard';
 import { SliderArrow } from './components/SliderArrow';
 import type { Slides } from './types/SliderTypes';
 import { slides } from './mocks/slidesData';
 import { ArrowDirection } from './constants/constants';
-import 'swiper/css';
+import 'swiper/swiper-bundle.css';
```
Comment (kerrimov) [inline src/components/Slider/Slider.tsx:10]: why do you import all bundle? Can we use only CSS for autoplay and navigation

Comment (PietyE) [reply src/components/Slider/Slider.tsx:10]: I don`t need css for auto-play and navigation modules. But I need styles for slider structure for this works. But in 6.0.2v of swiper we have only one css file to import :(  , so i had used it.

**Review by kerrimov — APPROVED** [review]

**Pull request merged**